### ISSUE TITLE
What to use instead of <marko-compiler-options preserve-comments> ?

### ISSUE THREAD

**Issue by eikaramba** [marko-js/marko#1425]
### Details

I'm compiling a MJML html document which heavily uses conditional tags for email rendering like
```
<!--[if !mso]><!-- -->
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<!--<![endif]-->
```
Normally marko is stripping these out, by using `<marko-compiler-options preserve-comments/>` everything works as expected. That is deprecated however and using `<html-comment>` is not giving the same result resp. it is also not feasible to wrap every occurance of comments with that as the code is auto generated by MJML

### Expected Behavior

keep conditional tags untouched by setting a global compiler configuration

### Actual Behavior

`<marko-compiler-options preserve-comments/>` works but is giving a deprecation warning.

`<html-comment>` is **WRONGLY** outputting:

```
<!--<!--[if !mso]><!-- --&gt; <meta http-equiv="X-UA-Compatible" content="IE=edge"> <!--<![endif]--&gt;-->
```

**Comment by DylanPiercey** [comment]
@eikaramba Marko actually does preserve the `[condition]` comments by default (even without the preserve comments enabled).

I could be wrong, but should the above not be:

```
<!--[if !mso]>
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<![endif]-->
```

Also `<html-comment>` just outputs a regular html comment based on the body content. It will escape the body content if needed as you are seeing, this is expected behavior. If you need multiple comments as it looks like you are trying to do, then you'll need multiple `<html-comment>` tags. They can not be nested.

**Comment by eikaramba** [comment]
The markup above is actual correct, that is because this is cross-compatible html code for various email clients. i know it looks stupid, but that is how one can target all those different email clients. MJML is doing that.


> Marko actually does preserve the [condition] comments by default (even without the preserve comments enabled).

Not in my experience, also not if i try it on the  "try live" page on the official marko webpage. It gets stripped out.
![grafik](https://user-images.githubusercontent.com/1320525/66853137-d9c51380-ef7e-11e9-8cc8-9145a206a0fa.png)

**Comment by DylanPiercey** [comment]
@eikaramba that makes sense. I think this is a bug on our end and I've put up a PR here: https://github.com/marko-js/marko/pull/1427

You can work around this using the `<html-comment>` tag like so:
```marko
<html-comment>[if !mso]><!-- </html-comment>
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<html-comment><![endif]</html-comment>
```

At least that seemed to work in my testing.

**Comment by DylanPiercey** [comment]
Released in `4.18.15`
